configure.ac

```diff
@@ -3408,6 +3408,31 @@ case "$OPT_ZSH_FPATH" in
     ;;
 esac
 
+dnl **********************************************************************
+dnl Check for fish completion path
+dnl **********************************************************************
+
+OPT_FISH_FPATH=default
+AC_ARG_WITH(fish-functions-dir,
+AC_HELP_STRING([--with-fish-functions-dir=PATH],[Install fish completions to PATH])
+AC_HELP_STRING([--without-fish-functions-dir],[Do not install fish completions]),
+  [OPT_FISH_FPATH=$withval])
+case "$OPT_FISH_FPATH" in
+  no)
+    dnl --without-fish-functions-dir option used
+    ;;
+  default|yes)
+    dnl --with-fish-functions-dir option used without path
+    FISH_FUNCTIONS_DIR="$datarootdir/fish/completions"
+    AC_SUBST(FISH_FUNCTIONS_DIR)
+    ;;
+  *)
+    dnl --with-fish-functions-dir option used with path
+    FISH_FUNCTIONS_DIR="$withval"
+    AC_SUBST(FISH_FUNCTIONS_DIR)
+    ;;
+esac
```

Comment:
This will get unwieldy when the third and fourth and .. and ... shell gets added. Is there a way to support the tab completion with a single configure flag invocation? `--with-shell-functions={fish|zsh}`?

Reply:
Maybe someone else with a better understanding of the build system (than me) can investigate (later on)?

Reply:
Yes, this can be deferred, potentially till when we want to add a third tab completion target even.